tests/unit/BackendClient-spec.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * Created by Ron.Yadgar on 11/3/2015.
+ */
+var proxyquire = require('proxyquire');
+var chai = require('chai');
+var expect = chai.expect;
+var sinon = require('sinon');
+var config = require('../../common/Configuration');
+var assert=require('assert')
+describe('Backend client facdefgstory spec', function() {
```

Comment:
1. Add a blank line after the requires for readability
2. describe name is gibberish